README.md

```diff
@@ -41,229 +42,300 @@ PID   USER     COMMAND
 / $
 ```
 
-Or you can specify the path to a JSON configuration file:
-```bash
-runc start config.json
-/ $ ps
-PID   USER     COMMAND
-1     daemon   sh
-5     daemon   sh
-/ $
-```
-Note: the use of the `start` command is required when specifying a 
-configuration file.
+### OCI Container JSON Format:
 
-### OCF Container JSON Format:
-
-Below is a sample `config.json` configuration file. It assumes that
+Below are sample `config.json` and `runtime.json` configuration files. It assumes that
```

Comment:
I think it makes sense to move this generic OCS documentation into opencontainers/specs.  Then you can link to that overview from opencontainers/runc, and not worry about keeping your OCS docs in sync with the upstream spec.  Folks suggesting changes to the spec would just update the docs when PRing their changes, and the OCS maintainers would review the whole change as a cohesive unit.